.github/workflows/test.yml

```diff
@@ -28,14 +30,34 @@ jobs:
           - laravel: 6.*
             testbench: 4.*
         exclude:
+          - laravel: 6.*
+            php: 8.1
+          - laravel: 6.*
+            php: 8.2
+          - laravel: 7.*
+            php: 8.1
+          - laravel: 7.*
+            php: 8.2
           - laravel: 8.*
             php: 7.2
+          - laravel: 8.*
+            php: 8.2
           - laravel: 9.*
             php: 7.2
           - laravel: 9.*
             php: 7.3
           - laravel: 9.*
             php: 7.4
+          - laravel: 9.*
+            php: 8.2
+          - laravel: 10.*
+            php: 7.2
+          - laravel: 10.*
+            php: 7.3
+          - laravel: 10.*
+            php: 7.4
+          - laravel: 10.*
+            php: 8.0
```

Comment:
Laravel 10 and php 8.2 missing? @rubenvanassche

Reply:
They should run since they're not on the excluded list, based upon https://laravel.com/docs/10.x/releases#support-policy